here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/IntHandlerForEventHandlers.java

```diff
@@ -48,6 +48,7 @@ public IntHandlerForEventHandlers(final @NotNull INaksha hub) {
         return reader.execute(rr);
       }
     } else if (request instanceof WriteRequest<?, ?> wr) {
+      // TODO check if storageId is provided
       try (final IWriteSession writer = nakshaHub().getAdminStorage().newWriteSession(ctx, true)) {
```

Comment:
Let's check the presence of mandatory field className. And related JUnit test as well, if it is found to be missing, returning bad request (400).
We need similar check for Storage as well, which I will add.